src/Servers/Kestrel/test/Interop.FunctionalTests/HttpClientHttp2InteropTests.cs

```diff
@@ -1189,6 +1189,7 @@ public async Task Settings_HeaderTableSize_CanBeReduced_Server(string scheme)
         // Nor does Kestrel yet support sending dynamic table updates, so there's nothing to test here. https://github.com/dotnet/aspnetcore/issues/4715
 
         [Theory]
+        [QuarantinedTest]
```

Comment:
I've also added this to https://github.com/dotnet/aspnetcore/pull/19568 , should merge OK.